source/BuildingBlocks.Infrastructure/FeatureFlag/FeatureFlagName.cs

```diff
@@ -54,4 +54,14 @@ public enum FeatureFlagName
     /// Whether to start using standard blob service client.
     /// </summary>
     UseStandardBlobServiceClient,
+
+    /// <summary>
+    /// Whether to enqueue BRS-023/027 messages via the Process Manager.
+    /// </summary>
+    EnqueueBrs023027MessagesFromProcessManager,
```

Comment:
If kept, follow the naming convention of the other feature flags:

```suggestion
    UseEnqueueBrs023027MessagesFromProcessManager,
```